usecases/config/environment.go

```diff
@@ -254,6 +254,18 @@ func FromEnv(config *Config) error {
 		config.QueryNestedCrossReferenceLimit = DefaultQueryNestedCrossReferenceLimit
 	}
 
+	if v := os.Getenv("QUERY_CROSS_REFERENCE_DEPTH_LIMIT"); v != "" {
+		limit, err := strconv.ParseInt(v, 10, 64)
+		if err != nil {
+			return fmt.Errorf("parse QUERY_CROSS_REFERENCE_DEPTH_LIMIT as int: %w", err)
+		} else if limit <= 0 {
+			limit = DefaultQueryCrossReferenceDepthLimit
```

Comment:
or you can return an error like we do it in other cases:
```
return fmt.Errorf("%s must be a positive value larger 0", varName)
```

Reply:
oh, we have a method `parsePositiveInt` for parsing int values, maybe we should use it?